docs/rfcs/0041-progressive-rollout.md

```diff
@@ -0,0 +1,291 @@
+# Summary
+
+# UI
+
+We'll introduce two kinds of UI, "Manual Setting" and "Template Setting".
+
+## Manual Setting
+
+Users can determine any percentage and schedule in this UI.
+
+![manual-setting-proto-ui](./images/0041-image1.png)
+
+## Template Setting
+
+Manual Setting has a lot of flexibility, but it’s hard when configuring a periodic increase in the percentage of a variation.
+In this case, Template Setting is useful.
+
+![template-setting-proto-ui](./images/0041-image2.png)
+
+## Progress bar of rollout
+
+![progress-bar-of-rollout](./images/0041-image4.png)
+
+# Important Notice
+
+* Users can use Progressive Rollout when the number of variations is equal to 2.
+* Users can't use same scheduled time in single auto ops rules. For example, users can not set true for 50% at 2023-01-01 00:06:00 and 80% at the same time.
+* The interval of time for each scheduled time must be at least 5 minutes.
+* Users can stop Progressive Rollout temporary.
```

Comment:
question:

When a user resumes a Progressive Rollout that has been stopped, how do we deal with Schedules that are already past their time?

Reply:
That's right. It seems to be that a temporary stop is impossible.

Reply:
Let me change it. Thank you.

Reply:
Talked at discord.